core/API/DataTableManipulator/Flattener.php

```diff
@@ -106,6 +106,18 @@ private function flattenRow
     (Row $row, $rowId, DataTable $dataTable, $level, $dimensionName,
                                 $labelPrefix = '', $parentLogo = false)
     {
+        $dimensions = $dataTable->getMetadata('dimensions');
```

Comment:
Any chance to do this maybe in the HtmlTable visualisation or so? Be nice to have this feature contained in one class or so and not spread across core, visualisations, etc.

Reply:
@sgiehl can you answer this comment when you have time?

Reply:
Not sure if there is a better/easier way to get all dimensions that were flattened. The only thing I can think of would be to get the report's metadata and then iterate through all sub reports to get all the dimensions. But not sure if that would work correctly with custom reports.
Nevertheless I think it might make sense to always have the `dimensions` metadata for flattened reports, as each dimension also has it's own metadata property. So the content of `dimensions` metadata makes it easy to identify the available dimension metadata properties.

Reply:
@tsteur can you check @sgiehl's reply?